### PR TITLE
Answered all the questions

### PR DESCRIPTION
**Pull request by sanket8397**
@sohmpatil Please review the answers in answers branch.

### CODE REVIEW

File: answer.txt [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+# virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
+hs_err_pid*
+
+
+2. When you try to merge one commit with a commit that can be reached by following the first commit’s history, Git simplifies things by moving the pointer forward because there is no divergent work to merge together – this is called a “fast-forward.”
+
+3. Git branches come in handy at a variety of points during the development of a project. Branches create a separate line of development from the main master branch. There are numerous benefits to doing so. Consider the following scenario: you are working on a project with your team and have completed a feature. You contact the client to request that they view the feature, but they are too busy, so you send them the project link.
+
+4. If you are working on a big project, then it is inevitable that you need to work with multiple repositories.
```
Comment (sohmpatil) [inline answer.txt]: @sanket8397 Please explain in detail.

File: answer.txt [hunk 2]
```diff
@@ -0,0 +1,33 @@
+1. The .gitignore file is a text file that tells Git which files or folders to ignore in a project.
```
Comment (sohmpatil) [inline answer.txt]: @sanket8397 Please add questions to all the answers.

**Review by sohmpatil — CHANGES_REQUESTED** [review]
@sanket8397 Complete the changes and then send it to review again.

File: answer.txt (hunk 2 above)
Comment (sanket8397) [reply answer.txt]: Done. Please review.

File: answer.txt (hunk 1 above)
Comment (sanket8397) [reply answer.txt]: Done. Please review.

**Review by sohmpatil — APPROVED** [review]
LGTM.

**Pull request merged**